src/hotspot/os/windows/sharedRuntimeRem.cpp

```diff
@@ -25,7 +25,6 @@
 #include "precompiled.hpp"
 #include "runtime/sharedRuntime.hpp"
 
-#ifdef _WIN64
```

Comment:
Just a heads up: Due to a bug, this entire file is never used at all

Reply:
I stand corrected: I forgot about Windows/ARM64. To correct myself: Due to a bug, this file, which is meant for Windows/x64, is used by Windows/ARM64 instead. The consequences of this are unknown

Reply:
What bug are you referring to that makes this file unused?

Reply:
https://mail.openjdk.org/pipermail/hotspot-dev/2024-October/095864.html

This file isn't unused, I misspoke. Instead, it is meant as the implementation of frem and drem for Windows x64, but due to a bug, it's potentially being wrongly used as the implementation of frem and drem for Windows/ARM64 instead

Reply:
Okay. I'll leave it to you to sort out that mess. :) But afaict from reading up on the discussion, this removal of `_WIN64` does not change anything in that respect, so I'll keep it.